pkg/roll/run_update.go

```diff
@@ -330,9 +336,13 @@ func (u *update) Run(ctx context.Context) (ret bool) {
 	hQuit := make(chan struct{})
 	defer close(hQuit)
 	watchDelay := 1 * time.Second
-	go u.hcheck.WatchService(string(newFields.Manifest.ID()), hChecks, hErrs, hQuit, watchDelay)
+	config := newFields.Manifest.GetConfig()
+	useHealthService, _ := config["use_health_service"].(bool)
```

Comment:
what if config["use_health_service"] isn't present? That's what the second return value is for.

Reply:
useHealthService defaults to false if it's not present.

Reply:
Are you sure we want this config to be in app pod manifests? it should probably be a setting on the roll farm right?

Reply:
I thought we test it with specific apps first but I'm open to making it a global config. What does @rudle think?

Reply:
Probably we need both, right? app specific so we can test applications like slug & p2-heartbeat and then roll farm wide so we can turn it on and off with the rest of the apps at Square.

Reply:
ah okay, I'm just out of the loop

Reply:
I would still check `ok` to make sure that the field is actually a bool, so you don't crash the server if someone has a typo

Reply:
The ok is not used and no panic occurs even if useHealthService is not a bool. See https://tour.golang.org/methods/15, "If not, ok will be false and t will be the zero value of type T, and no panic occurs."

It's only when you do
```golang
useHealthService := config["use_health_service"].(bool)
```
for it to panic when the value is of type not bool.

Reply:
The question is, what do you want to do if someone _intended_ to set it to true, but messed up up with a typo, e.g. `truee`. You're going to treat it as false. Perhaps that's fine, but it's worth thinking about

Reply:
I do feel better that it at least won't crash the server, probably for a flag like this it's okay to not introduce errors here which will be difficult to recover from

Reply:
I'd think I want only `true` to be treated as true and everything else be false otherwise there needs to be code to interpret what the user implied which is I don't think can be fully correct.

Reply:
You can check out https://golang.org/pkg/strconv/#ParseBool to error on non-booleans